/opt/homebrew/anaconda3/envs/myconda-3.11/lib/python3.11/site-packages/whisper/__init__.py:69: UserWarning: /Users/hoyoul/.cache/whisper/base.pt exists, but the SHA256 checksum does not match; re-downloading the file
  warnings.warn(
  0%|                                               | 0.00/139M [00:00<?, ?iB/s]  2%|▊                                     | 2.95M/139M [00:00<00:04, 29.8MiB/s]  5%|██                                    | 7.45M/139M [00:00<00:03, 39.8MiB/s]  9%|███▌                                  | 12.9M/139M [00:00<00:02, 46.5MiB/s] 15%|█████▌                                | 20.5M/139M [00:00<00:02, 52.8MiB/s] 21%|███████▉                              | 29.1M/139M [00:00<00:01, 65.3MiB/s] 26%|█████████▋                            | 35.4M/139M [00:00<00:01, 64.3MiB/s] 30%|███████████▍                          | 41.6M/139M [00:00<00:01, 63.7MiB/s] 34%|█████████████                         | 47.7M/139M [00:00<00:01, 63.2MiB/s] 39%|██████████████▋                       | 53.8M/139M [00:00<00:01, 63.0MiB/s] 43%|████████████████▍                     | 59.8M/139M [00:01<00:01, 62.8MiB/s] 47%|██████████████████                    | 65.8M/139M [00:01<00:01, 62.6MiB/s] 52%|███████████████████▋                  | 71.8M/139M [00:01<00:01, 62.5MiB/s] 56%|█████████████████████▎                | 77.7M/139M [00:01<00:01, 62.5MiB/s] 60%|██████████████████████▉               | 83.7M/139M [00:01<00:00, 62.5MiB/s] 65%|████████████████████████▌             | 89.6M/139M [00:01<00:00, 62.4MiB/s] 69%|██████████████████████████▏           | 95.6M/139M [00:01<00:00, 62.4MiB/s] 73%|████████████████████████████▌          | 102M/139M [00:01<00:00, 62.3MiB/s] 78%|██████████████████████████████▎        | 108M/139M [00:01<00:00, 62.3MiB/s] 82%|███████████████████████████████▉       | 113M/139M [00:01<00:00, 62.3MiB/s] 86%|█████████████████████████████████▌     | 119M/139M [00:02<00:00, 62.3MiB/s] 90%|███████████████████████████████████▎   | 125M/139M [00:02<00:00, 62.3MiB/s] 95%|████████████████████████████████████▉  | 131M/139M [00:02<00:00, 62.3MiB/s] 99%|██████████████████████████████████████▋| 137M/139M [00:02<00:00, 62.3MiB/s]100%|███████████████████████████████████████| 139M/139M [00:02<00:00, 60.9MiB/s]
 * Serving Flask app 'server'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:5000
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 136-966-870
 * Serving Flask app 'server'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
On macOS, try disabling the 'AirPlay Receiver' service from System Preferences -> General -> AirDrop & Handoff.
/opt/homebrew/anaconda3/envs/myconda-3.11/lib/python3.11/multiprocessing/resource_tracker.py:254: UserWarning: resource_tracker: There appear to be 2 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
/opt/homebrew/anaconda3/envs/myconda-3.11/lib/python3.11/multiprocessing/resource_tracker.py:254: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
 * Serving Flask app 'server'
 * Debug mode: on
Address already in use
Port 5000 is in use by another program. Either identify and stop that program, or start the server with a different port.
On macOS, try disabling the 'AirPlay Receiver' service from System Preferences -> General -> AirDrop & Handoff.
 * Serving Flask app 'server'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://125.243.128.71:5001
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 136-966-870
/opt/homebrew/anaconda3/envs/myconda-3.11/lib/python3.11/multiprocessing/resource_tracker.py:254: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
/opt/homebrew/anaconda3/envs/myconda-3.11/lib/python3.11/multiprocessing/resource_tracker.py:254: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
 * Serving Flask app 'server'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://125.243.128.71:5001
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 136-966-870
/opt/homebrew/anaconda3/envs/myconda-3.11/lib/python3.11/multiprocessing/resource_tracker.py:254: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
/opt/homebrew/anaconda3/envs/myconda-3.11/lib/python3.11/multiprocessing/resource_tracker.py:254: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
 * Serving Flask app 'server'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://125.243.128.71:5001
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 136-966-870
/opt/homebrew/anaconda3/envs/myconda-3.11/lib/python3.11/multiprocessing/resource_tracker.py:254: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
/opt/homebrew/anaconda3/envs/myconda-3.11/lib/python3.11/multiprocessing/resource_tracker.py:254: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
